### PR TITLE
Bump admin gem to 4.0.1

### PR DESCRIPTION
**Pull request by fofr**
* Allow environment_style to be set to integration
* Use admin template config for app title

Some of the earlier dependencies on `environment_style` were refactored out in #444

cc @boffbowsh

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<% content_for :app_title, 'GOV.UK Signon' %>
 <% content_for :page_title, "#{yield(:title)} | GOV.UK Signon" %>
```
Comment (benilovj) [inline app/views/layouts/application.html.erb]: Should the page title still have the app name?

Comment (fofr) [inline app/views/layouts/application.html.erb]: At the moment there isn't a pattern for doing this automatically. But I think there should be.

Comment (benlovell) [inline app/views/layouts/application.html.erb]: @fofr will you act on this or should we get this merged?

Comment (fofr) [inline app/views/layouts/application.html.erb]: I wasn't going to alter it as part of this PR. I think it's something that's needed in the admin template itself.

**Pull request merged**